pkg/operator/ceph/client/controller.go

```diff
@@ -325,12 +325,9 @@ func (r *ReconcileCephClient) createOrUpdateClient(cephClient *cephv1.CephClient
 		},
 		StringData: map[string]string{
 			cephClient.Name: key,
-			// CSI requires userID and userKey for RBD
+			// CSI requires userID and userKey in secret
 			"userID":  cephClient.Name,
 			"userKey": key,
```

Comment:
We can directly remove, we don't need think about backport compatibility or this will not affect upgrade?

Reply:
No problem with upgrade and backward compatibility https://github.com/rook/rook/issues/16175#issuecomment-3131094573